utils/utils_test.go

```diff
@@ -167,3 +169,65 @@ func TestIsWildcardParentheses(t *testing.T) {
 		t.Errorf("TestIsWildcardParentheses() == %t, want %t", got, want)
 	}
 }
+
+func TestAntPathToRegExp(t *testing.T) {
+	var fileSystemPaths []string = []string{
+		"dev/a/b.txt",
```

Comment:
Let's also add the following path, and see that the logic ant pattern translation works well:
"dev/a1/a2/b.txt"